src/vmx.rs

```diff
@@ -798,17 +825,15 @@ fn prepare_vmx_memory_region(vmx_region: *mut u8, vmx_region_size: usize) {
 
     for i in mem::size_of::<u32>()..vmx_region_size {
         unsafe {
-            *vmx_region.offset(i as isize) = 0;
+            vmx_region.add(i).write(0);
```

Comment:
This change is good, but there's probably a better way to write "memset(&vmx_region[sizeof(u32)], 0, PAGE_SIZE-sizeof(u32))" in rust these days. This is a note to self mostly, no need to change anything.

Reply:
Thank you for notifying this, I found that there is `core::ptr::write_bytes` which handles this better.